src/api/app/controllers/webui/request_controller.rb

```diff
@@ -11,20 +11,19 @@ class Webui::RequestController < Webui::WebuiController
   after_action :verify_authorized, only: [:create]
 
   def create
-    request = BsRequest.new(bs_request_params)
-    authorize request, :create?
-    begin
-      request.save!
-    rescue APIError => e
-      flash[:error] = e.message
+    @request = BsRequest.new(bs_request_params)
```

Comment:
This also allows us to access the newly created request in an `after_action` callback. One example in which this is useful is when we need to supersede one/multiple request(s). We first need to create a request, then mark the superseded requests as such by the newly created request.

Reply:
`request` is already used by rails in the controller, as `response`. I would avoid to call it `@request`, even if ugly `@bs_request` would be more appropriate